### ISSUE TITLE
Bug with batch norm

### ISSUE THREAD

**Issue by yxliu-ntu** [NVIDIA/caffe#436]
If we use "store_blobs_in_old_format:True" to store weights trained by nvidia-caffe and read these weights in BVLC-caffe, it will cause some problem and output wrong values.
After parsing the source code, I find that it is because Nvidia-Caffe implement the BatchNorm layer in a different way. It stores the real batch norm 'variance' and 'mean' value in the first 2 blobs in weight file and store '0' in  the 3rd blob and uses variance and mean direactly when inference,  while BVLC-caffe doesn't. (BVLC needs values stored in 3rd blob to calculate the real variance and mean)

If we reset the values in the 3rd blob of BatchNrom layer from "0" to "1", then everything works well again.

**Comment by drnikolaev** [comment]
Hi @Mithril-NTU thank you. We noticed this too and fix is coming in next release

**Comment by yxliu-ntu** [comment]
Thank you very much!

**Comment by Soledad89** [comment]
Hi, @Mithril-NTU, I run into the same error. Can you share your modified codes to solve this issue? Thanks so much.

**Comment by bonseyes-admin** [comment]
@drnikolaev Do you have an ETA on the next release date?

**Comment by drnikolaev** [comment]
Please check v0.16.5 and reopen the issue if the problem still exists.
